docs/integrations/sekoiaio_activity_logs.md

```diff
@@ -0,0 +1,12 @@
+uuid: 41e3ca4e-a714-41aa-ad69-684a0b3835fc
+name: SEKOIA.IO activity logs
+
+## Overview
+SEKOIAIO.IO activity logs collect all operations done, on SEKOIA.IO, by the members of the community. It helps to monitor activities and detect malicious behavior.
+
```

Comment:
Hello @squioc 
Could you please add a liste of 10 to 15 relevant actions loged that are interesting from a security point of view?